### DISCUSSION TITLE
It is possible to run it on Windows server 2022

### DISCUSSION THREAD

**Discussion by dezordo22dogs** [grizzlypeak3d/DJV#506] [Q&A]
Hi to all,

i have received te following request:

My collegues ask me to run DJV on Windows server 2022

There are any particular requirements?

It is need to be build as indicated here https://darbyjohnston.github.io/DJV/build_windows.html?

**Comment by darbyjohnston** [comment]
Hi, and sorry for the long delay; you can find ZIP files here that should work:
https://github.com/darbyjohnston/DJV/releases/

DJV requires OpenGL support.
